benchmarks/scripts/ec2_script.sh

```diff
@@ -0,0 +1,12 @@
+export INSTANCE_TYPE=$(curl -s http://169.254.169.254/latest/meta-data/instance-type)
```

Comment:
This is a very bad thing to direct expose ip address in github.

Reply:
This is a public feature of aws ec2 instance. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html